### PR TITLE
Update to v1.2.0

### PR DESCRIPTION
**Pull request by bermeitinger-b**

### CODE REVIEW

**Comment by bermeitinger-b** [comment]
bot, build

File: org.signal.Signal.appdata.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@
     </screenshot>
   </screenshots>
   <releases>
+  	<release version="1.2.0" date="2018-01-24"/>
```
Comment (nedrichards) [inline org.signal.Signal.appdata.xml]: indenting here?

**Comment by Barthalion** [comment]
Looks like git history exploded so it'll be probably better to cherry-pick changes to fresh clone/master branch and git push directly.

**Comment by bermeitinger-b** [comment]
Seems like the appstream.xml doesn't need changes anyhow. The other PR went fine and updated correctly.

**Pull request closed without merging**